engine/src/cubos/engine/imgui/plugin.cpp

```diff
@@ -41,6 +42,8 @@ void cubos::engine::imguiPlugin(Cubos& cubos)
     cubos.addPlugin(windowPlugin);
     cubos.addPlugin(rendererPlugin);
 
+    cubos.addResource<DataInspector>();
```

Comment:
You should also add this resource to the imgui plugin docs on the `## Resources` section (see other plugins)